.travis/translate.py

```diff
@@ -0,0 +1,112 @@
+import json, os, sys
+#
+# This script is intended to search for keywords in english and translate to other languages.
+# Please be very careful when using certain keywords in the dicctionary, because this script
+# can search-replace code which will cause the form to break.
+#
+
+
+#   The envisioned use would be something like this:
+#       $ python .travis/translate.py  "./public/js/app.bundle.js"   "./local/es.json"
+#       
+
+translation_file = "app.bundle.js"
+json_file = "local/complaintDict.json"
+directory = os.path.dirname(os.path.abspath(__file__))
+
+
+
+#
+# A sample json file looks like this, but notice they are stored in the local directory
+#
+
+sample_dict = {
```

Comment:
Could we move this sample dict file out into it's own .json file?

Reply:
Ah, yes that was there to explain the json format only. It was not being used. The dict has been put in different paths and it has been tested.